### DISCUSSION TITLE
Need more contributors!

### DISCUSSION THREAD

**Discussion by SivaaB** [AhmedThahir/AhmedThahir.github.io#105] [Announcements]
Need more people auditing the notes, and suggesting edits. 

I'm adding y'all as contributors, hoping that you guys can go through the notes taken so far and improve them whenever possible. I'm not asking you to generate "new" notes just for this purpose (that may take up a lot of your time).

By "improve", I mean if you identify a disparity between what's being taught in the lecture and the notes already taken on the project, make a pull request to bridge this gap.

**Comment by MurluKrishna4352** [comment]
I'll ask more people to join in

**Comment by AhmedThahir** [comment]
Any updates?

**Comment by syphxncodes** [comment]
i have checked math 1 notes for polar coordinates, seems to be fine, I wanted to know where can I find MOW notes?

**Comment by AhmedThahir** [comment]
Any updates @SivaaB ?

**Comment by AhmedThahir** [comment]
Closing this as stale.
